pkg/cli/presenter/vmclarity.go

```diff
@@ -88,7 +88,7 @@ func (v *VMClarityPresenter) ExportSbomResult(ctx context.Context, res families.
 		assetScan.Stats = &models.AssetScanStats{}
 	}
 
-	errs := []string{}
+	var errs []string
```

Comment:
Would be nice to have a separate PR for this fix.

Reply:
this fix shouldn't be in this PR, it was pulled out into a separate PR and so far has been rejected because of the nullable issues. I'll remove this.